api/internal.yaml.template

```diff
@@ -25,12 +25,241 @@ paths:
+          description: The assertion type (SAML / OIDC)
+          required: true
+          schema:
+            $ref: "#/components/schemas/AssertionType"
+        - name: x-pagopa-lollipop-user-id
+          in: header
+          description: User's fiscal code.
+          required: true
+          schema:
+            $ref: "#/components/schemas/LollipopUserId"
+        - name: x-pagopa-lollipop-public-key
+          in: header
+          description: User's public key
+          required: true
+          schema:
+            $ref: "#/components/schemas/LollipopPublicKey"
+        - name: signature-input
+          in: header
+          description: The signature input, needed to verify the `signature` header
+          required: true
+          schema:
+            $ref: "#/components/schemas/LollipopSignatureInput"
+        - name: signature
+          in: header
+          description: The signature of the HTTP request, signed by the client with its private key.
+          required: true
+          schema:
+            $ref: "#/components/schemas/LollipopSignature"
+      responses:
+        '200':
+          description: "Success response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FastLoginResponse"
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BadRequest"
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Unauthorized"
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/InternalServerError"
 components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Functions-Key
   schemas:
+    InternalServerError:
+      description: InternalServerError
+      allOf:
+        - $ref: "#/components/schemas/ProblemDetail"
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum: [500]
+              example: 500
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
+    BadRequest:
+      description: Bad Request
+      allOf:
+        - $ref: "#/components/schemas/ProblemDetail"
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum: [400]
+              example: 400
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
+    Unauthorized:
+      description: Bad Request
```

Comment:
```suggestion
    Unauthorized:
      description: Unauthorized
```

Reply:
nice catch. solved in https://github.com/pagopa/io-functions-fast-login/pull/8/commits/3fd6b7def8fe64e52fb5da73e2043a6612eeb066